File: CoWorkHub/UI/coworkhub_desktop/lib/models/country.dart
import 'package:json_annotation/json_annotation.dart';
part 'country.g.dart';

@JsonSerializable()
class Country {
  final int countryId;
  final String countryName;
  final bool? isDeleted;

  Country({required this.countryId, required this.countryName, this.isDeleted});

  factory Country.fromJson(Map<String, dynamic> json) {
    return Country(
      countryId: json['countryId'],
      countryName: json['countryName'],
      isDeleted: json['isDeleted'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'countryId': countryId,
      'countryName': countryName,
      'isDeleted': isDeleted,
    };
  }
}

File: CoWorkHub/CoWorkHub.Model/Country.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CoWorkHub.Model
{
    public class Country
    {
        public int CountryId { get; set; }
        public string CountryName { get; set; }
        public bool IsDeleted { get; set; } = false;
    }
}
